libmscore/score.h

```diff
@@ -529,6 +529,10 @@ class Score : public QObject, public ScoreElement {
       void cmdAddOttava(Ottava::Type);
       void cmdAddStretch(qreal);
       void cmdResetNoteAndRestGroupings();
+      void cmdDoubleDuration()      { cmdIncDecDuration(-1, 0); }
+      void cmdHalfDuration()        { cmdIncDecDuration( 1, 0); }
+      void cmdIncDurationDotted()   { cmdIncDecDuration(-1, 1); }
+      void cmdDecDurationDotted()   { cmdIncDecDuration( 1, 1); }
```

Comment:
Actually I'm thinking of eventually changing that parameter into a char or enum and be able to "step Double-Dotted" (Maybe Ctrl-Shift), "step Triple-dotted", and "step quad-dotted".